.gitlab-ci.yml

```diff
@@ -696,6 +705,7 @@ trigger-simnet:
     - if: $CI_PIPELINE_SOURCE == "pipeline"
       when: never
     - if: $CI_PIPELINE_SOURCE == "web" && $CI_COMMIT_REF_NAME == "master"
+    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
```

Comment:
we did not agree to run simnet from substrate on PRs, did it change somewhere?

Reply:
I think the earlier we catch a bug the better ; let's discuss this in simnet team channel

Reply:
maybe replace with
`    - if: $CI_COMMIT_MESSAGE  =~ /.*build-for-simnet.*/`

Reply:
let's keep it on "revert me" basis for now